dxr/plugins/xbl/__init__.py

```diff
@@ -29,11 +29,13 @@ def __init__(self, path, contents, plugin_name, tree):
     @property
     def analyzer(self):
         if not self._analyzer:
-            self._analyzer = XBLAnalyzer(self.tree, self.contents, self.tree.source_encoding)
+            self._analyzer = XBLAnalyzer(self.path, self.tree,
+                                         self.contents, self.tree.source_encoding)
         return self._analyzer
 
     def is_interesting(self):
-        return "<bindings" in self.contents and super(FileToIndex, self).is_interesting()
+        return (self.path.endswith('.xml') and '<bindings' in self.contents
```

Comment:
It's academic the way things currently stand (but things change): we should probably put the super() call first, since it's cheapest. It's the thing that will skip binary files if we started reading them in again for some reason and if one ends up having an .xml extension due to changes in our definition of what a binary file is, etc.